.github/workflows/pytest.yaml

```diff
@@ -27,7 +27,12 @@ jobs:
           rosdep update
           rosdep install -i --skip-keys "caret_analyze_cpp_impl" --from-paths . -y --rosdistro humble
       - name: Install dependent packages
-        run: python3 -m pip install -r src/requirements.txt
+        run: |
+          python3 -m pip install -r src/requirements.txt
+          if dpkg -l | grep -q python3-typeguard; then
+            sudo apt remove -y python3-typeguard
+            pip install typeguard
```

Comment:
`typeguard` is not used in caret and does not need to be installed again.

Reply:
Fixed in https://github.com/tier4/caret_analyze/pull/529/commits/7feb2594a54e2272e94986296144de65e524f2e7.